packages/driver/cypress/integration/e2e/multi-domain/multi_domain_spec.ts

```diff
@@ -275,4 +275,20 @@ describe('domain validation', { experimentalSessionSupport: true }, () => {
     .invoke('text')
     .should('equal', 'Welcome TJohnson')
   })
+
+  it('creates a spec bridge for https://example.com', () => {
+    cy.visit('/fixtures/auth/index.html') // Establishes Primary Domain
+    cy.switchToDomain('example.com', () => {
+      cy.visit('https://example.com')
```

Comment:
Suggest using one of the hosts in the `cypress.json` file instead of loading a real page.

Reply:
can do, i didn't realize you needed a different port for https. Will update!